examples/slu/slurp/data_utils/decode_resample.py

```diff
@@ -0,0 +1,114 @@
+# ! /usr/bin/python
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import json
+import multiprocessing
+import os
+from pathlib import Path
+
+import sox
+import tqdm
+from sox import Transformer
+from tqdm.contrib.concurrent import process_map
+
+FIELD_AUDIO = "audio_filepath"
+FIELD_TEXT = "text"
+FIELD_DATA_DIR = "data_dir"
+
+
+def process(x):
+    if not isinstance(x[FIELD_TEXT], str):
+        x[FIELD_TEXT] = ''
+    else:
+        x[FIELD_TEXT] = x[FIELD_TEXT].lower().strip()
+
+    data_dir = x[FIELD_DATA_DIR]
+    input_file = Path(x[FIELD_AUDIO])
+    if not input_file.is_absolute():
+        input_file_path = data_dir / input_file
+    else:
+        input_file_path = input_file
+
+    output_file = Path(input_file.stem + ".wav")
+
+    if "slurp_real" in str(input_file_path):
+        output_dir = Path("wavs/slurp_real")
+    else:
+        output_dir = Path("wavs/slurp_synth")
+
+    output_file_path = str(data_dir / output_dir / output_file)
+
+    if not os.path.exists(output_file_path):
+        tfm = Transformer()
```

Comment:
Dont use sox transformer if possible, use librosa or ffmpeg if possible, it is safer.